pkg/controllers/controlplanemachinesetgenerator/nutanix.go

```diff
@@ -66,3 +97,40 @@ func buildControlPlaneMachineSetNutanixMachineSpec(logger logr.Logger, machines
 
 	return msac, nil
 }
+
+// buildNutanixFailureDomains builds a Nutanix flavored FailureDomains for the ControlPlaneMachineSet.
+func buildNutanixFailureDomains(logger logr.Logger, failureDomains *failuredomain.Set, infrastructure *configv1.Infrastructure) (machinev1.FailureDomains, error) {
+	logger.V(4).Info("to build Nutanix failureDomain for ControlPlaneMachineSet: ", "input failure domains: ", failureDomains.List())
```

Comment:
Nit, this is structured logging not concatenation
```suggestion
	logger.V(4).Info("building Nutanix failureDomains for ControlPlaneMachineSet", "baseFailureDomains", failureDomains.List())
```

Reply:
Changed.